kyber_gt.go

```diff
@@ -67,18 +69,20 @@ func (k *KyberGT) Add(a, b kyber.Point) kyber.Point {
 }
 
 func (k *KyberGT) Sub(a, b kyber.Point) kyber.Point {
-	aa := a.(*KyberGT)
-	bb := b.(*KyberGT)
-	bls12381.NewGT().Sub(k.f, aa.f, bb.f)
-	return k
+	panic("not available")
 }
```

Comment:
Why not implement subtraction as being the multiplication by the inverse if we are using it as a multiplicative group?
I'm not too fond of panics in production code, but I guess it could work.

Reply:
Right, good idea, let's do this.
I actually see that the kilic library has been updated quite some since I last made this wrapper as well and there is now a `Fr` type (Scalar) fully defined, instead of using big.Int so also for a future todo to use this in the wrapper.

Reply:
https://github.com/drand/kyber-bls12381/issues/13